docs/examples/KcardGrid/BasicGrids/1-1-1.vue

```diff
@@ -0,0 +1,76 @@
+<template>
```

Comment:
Can we name the folder to KCardGrid to ensure proper TitleCase naming convention?